### PR TITLE
Start testing with playwright

### PR DESCRIPTION
**Pull request by lirbre**
Brings the cmdk config and write a initial test just to validate if I should stick w/ that kinda of testing

### CODE REVIEW

**Comment by lirbre** [comment]
> This works! You could also create a separate test folder with a next app and create various tests scenarios there.

Got it! I added a test that is actually working. I'll be adding more tests time to time. Maybe in another PRs to make easier to keep a standard.

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    baseURL: 'http://localhost:3000',
+  },
+  timeout: 5000,
+  webServer: {
+    command: 'npm run dev',
+    url: 'http://localhost:3000',
+    cwd: './test',
+    reuseExistingServer: !process.env.CI,
+  },
+  projects: [
```
Comment (guilbill) [inline playwright.config.ts:19]: Why not testing for other browsers ?

Comment (lirbre) [reply playwright.config.ts:19]: tbh I just pushed some config from another project. I was just trying to bring some initial config to start writing some tests.

which browsers should we be adding?

**Comment by emilkowalski** [comment]
Thank you for this @lirbre. I decided to go with a different approach by creating a test directory with a next app inside so we can create pages for various testing scenarios. Here's th PR if you are curious - https://github.com/emilkowalski/sonner/pull/10.

Once again, I really appreciate this PR and apologies for not merging it.

**Pull request closed without merging**